examples/vanilla-es6/src/app.js

```diff
@@ -0,0 +1,34 @@
+import Controller from './controller';
+import * as helpers from './helpers';
+import Template from './template';
+import Store from './store';
+import Model from './model';
+import View from './view';
+
+let todo;
```

Comment:
Is this global mutable variable with an implicit lifecycle really necessary? Why can't we call the `setView` directly on load?